test/bats/test_helper.bash

```diff
@@ -3,7 +3,7 @@ _common_setup() {
     bats_load_library bats-support
     bats_load_library bats-assert
     bats_load_library bats-file
-    bats_load_library bats-detik/detik.bash
+    bats_load_library bats-detik/lib/detik.bash
```

Comment:
please revert this change :)

there is no lib in the action files or nix package

Reply:
I just saw this comment after I tried fixing it by upgrading bats-core/bats-action to v3.0.0 which upgrades bats-detik. I think bats-detik reorganized so the entry point for the lib is at bats-detik/lib/detik.bash.

Would you prefer to revert the upgrade to the action and then revert the lib path?

Reply:
first of all thanks for the change :)

but not it's no a version issue, it's the fact that for both the action and the nix package the lib folder of bats-detik is moved under bats-detik dir itself for reasons :) (I know it 'cause I'm the maintainer of both the action and the nix package)

locally yes if you are not using the nix flake to quickly spawn the dev env for this project you need to change that load statement

Reply:
I see. I'll revert the relevant changes then.

Reply:
(so yes you need to put it back to `bats_load_library bats-detik/detik.bash` to make it run here.
Or if you want you can add an if statement that will check for the lib folder and load the right file based on the presence, so it can be run locally without flake and with